tests/unit/command/test_imp_url.py

```diff
@@ -44,3 +49,57 @@ def test_import_url_no_exec(mocker):
     assert cmd.run() == 0
 
     m.assert_called_once_with("src", out="out", fname="file", no_exec=True)
+
+
+def test_import_url_https(dvc):
+    cli_args = parse_args(
+        ["import-url", "https://data.dvc.org/get-started/data.xml"]
+    )
+    assert cli_args.func == CmdImportUrl
+
+    cmd = cli_args.func(cli_args)
+    assert cmd.run() == 0
```

Comment:
Not clear how this is related to this change. Also, this is not really a unit test.

Reply:
I added the test to understand the behavior of the code for this case. I decided to keep it, so another functionality could keep being tested.

Reply:
But we don't need this. We have https://github.com/iterative/dvc/blob/2f620e9226f6a337cd04da9748b4f40cd19efd1f/tests/func/test_import_url.py#L131 already, which, unlike this test, doesn't rely on external resources at all. Plus it is absolutely unrelated to this PR, making it harder to review. Let's remove it.

Reply:
Sure. Done!